api/src/test/java/org/openmrs/api/CohortServiceTest.java

```diff
@@ -440,9 +444,9 @@ public void getCohortsContainingPatient_shouldNotReturnVoidedCohorts() throws Ex
 	public void getCohortsContainingPatient_shouldReturnCohortsThatHaveGivenPatient() throws Exception {
```

Comment:
Yes, this is the one that is failing. Membership is getting saved to the database, since the ID got generated.